core/templates/third-party-imports/midi-js.import.ts

```diff
@@ -16,12 +16,12 @@
  * @fileoverview This file imports the MIDI.js library.
  */
 
-require('static/midi-js-c26ebb/build/MIDI.js');
-window.Base64Binary = require('static/midi-js-c26ebb/inc/shim/Base64binary.js');
+import 'midi/build/MIDI.js';
+window.Base64Binary = require('/node_modules/midi/inc/shim/Base64binary.js');
```

Comment:
Having node_modules in the require statement is not a convention we follow. Please explain why you need this and why can we do `import * as 'midi'` from midi or something along that lines.

Reply:
correct, we dont want `node_modules` within the path